src/main/java/acme/entities/notices/Notice.java

```diff
@@ -0,0 +1,56 @@
+
+package acme.entities.notices;
+
+import java.util.Date;
+
+import javax.persistence.Entity;
+import javax.persistence.Temporal;
+import javax.persistence.TemporalType;
+import javax.validation.constraints.Email;
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Past;
+
+import org.hibernate.validator.constraints.Length;
+import org.hibernate.validator.constraints.URL;
+
+import acme.client.data.AbstractEntity;
+import lombok.Getter;
+import lombok.Setter;
+
+@Entity
+@Getter
+@Setter
+public class Notice extends AbstractEntity {
+
+	//Serialisation identifier -----------------------------------------------------------------
+
+	private static final long	serialVersionUID	= 1L;
+
+	//Attributes --------------------------------------------------------------------------------
+
+	@Past
+	@NotNull
+	@Temporal(TemporalType.TIMESTAMP)
+	private Date				instantiationMoment;
+
+	@NotBlank
+	@Length(max = 75)
+	private String				title;
+
+	@NotBlank
+	@Length(max = 75)
+	// TODO: The author must be computed as follows: “〈username〉 - 〈surname, name〉”
+	private String				author;
+
+	@NotBlank
+	@Length(max = 100)
+	private String				message;
+
+	@Email
```

Comment:
Add length constraints (between 0 and 255 according to on your tutorials, in url and email fields)
[https://ev.us.es/webapps/discussionboard/do/message?action=list_messages&course_id=_85092_1&nav=discussion_board_entry&conf_id=_405265_1&forum_id=_234042_1&message_id=_405561_1](url)